go.mod

```diff
@@ -28,7 +28,7 @@ require (
 	golang.org/x/time v0.0.0-20191024005414-555d28b269f0 // indirect
 	google.golang.org/appengine v1.6.5 // indirect
 	google.golang.org/genproto v0.0.0-20200413115906-b5235f65be36 // indirect
-	google.golang.org/grpc v1.26.0
+	google.golang.org/grpc v1.27.0
```

Comment:
is this change expected?

Reply:
Ah, no, it seems to be an issue in the master branch. The `go mod verify` 'check' that exists does not detect an error, it modifies the file...

Reply:
Removed, will send an other PR to correct the `make mod-check` test and have it report an error.

Reply:
See #961